### PR TITLE
Add a doc of deploying onecloud on minikube to take a glance

### PR DESCRIPTION
**Pull request by lshmouse**
OneCloud is a very good production for multi cloud management. But the all-in-one deployment is too complicated. So I try to start the OneCloud on the minikube to take a glance at it. I hope this doc will be useful to other OneCloud users and developers. Thanks~

For lack of experience of OneCloud, some content may be not convenient and suggestions will be welcomed~

### CODE REVIEW

File: content/zh/docs/quickstart/minikube.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+```
+
+登陆webUI
+```bash
+kubectl -n onecloud port-forward `kubectl -n onecloud get pods  | grep "example-onecloud-cluster-web"| cut -f1 -d" "` 9999:443
+```
+打开浏览器：https://localhost:9999 
+
+### 带解决的问题
+4类Pod启动失败，问题还在分析中，但不影响体验onecloud
```
Comment (zexi) [inline content/zh/docs/quickstart/minikube.md:88]: 用 minikube 来运行 all in one 的想法很赞 ;)

host-deployer 启动失败应该是 minikube 没有使用我们的内核，该服务需要 nbd 内核模块。
可以参考 [安装 onecloud 依赖内核](https://docs.yunion.io/docs/setup/controlplane/#%E5%AE%89%E8%A3%85-onecloud-%E4%BE%9D%E8%B5%96%E5%86%85%E6%A0%B8) 安装我们的内核。

另外能把以下 ４ 个 pod 启动失败的日志发下吗？

Comment (lshmouse) [reply content/zh/docs/quickstart/minikube.md:88]: - example-onecloud-cluster-host-deployer
  (combined from similar events): Unable to attach or mount volumes: unmounted volumes=[cloud etc-yunion], unattached volumes=[run default-token-bfcsk cloud var-log-openvswitch sys var-run-openvswitch usr proc certs var-run-vmware config dev tmp etc-yunion]: timed out waiting for the condition
  MountVolume.SetUp failed for volume "etc-yunion" : hostPath type check failed: /etc/yunion is not a directory
- example-onecloud-cluster-notify
 containers with unready status: [feishu feishu-robot dingtalk-robot]
- example-onecloud-cluster-autoupdate
  containers with unready status: [autoupdate]
  ```
  [I 200529 17:06:23 ???] Auth complete!!
  [I 200529 17:06:23 ???] PolicyManager start to fetch policies ...
  [F 200529 17:06:23 ???] Init kubernetes driver: failed to get onecloud cluster onecloud/default: onecloudclusters.onecloud.yunion.io "default" not found
  ```
- example-onecloud-cluster-monitor
  Back-off pulling image "registry.cn-beijing.aliyuncs.com/yunionio/monitor:latest"
  Error: ImagePullBackOff

Comment (lshmouse) [reply content/zh/docs/quickstart/minikube.md:88]: @zexi 有开发者交流群吗，也是刚开始学习onecloud，希望请教一下～

Comment (lshmouse) [reply content/zh/docs/quickstart/minikube.md:88]: - example-onecloud-cluster-notify
          message: >-
            OCI runtime create failed: container_linux.go:346: starting
            container process caused "exec: \"/opt/yunion/bin/feishu\": stat
            /opt/yunion/bin/feishu: no such file or directory": unknown
          startedAt: '2020-05-29T10:35:15Z'

Comment (zexi) [reply content/zh/docs/quickstart/minikube.md:88]: > @zexi 有开发者交流群吗，也是刚开始学习onecloud，希望请教一下～

麻烦告知下微信号，我拉你进群

> * example-onecloud-cluster-notify
>   message: >-
>   OCI runtime create failed: container_linux.go:346: starting
>   container process caused "exec: "/opt/yunion/bin/feishu": stat
>   /opt/yunion/bin/feishu: no such file or directory": unknown
>   startedAt: '2020-05-29T10:35:15Z'

这个问题应该是照这个文档安装的 operator 是 dev 版本的，没有指定组件的统一版本，导致有些服务镜像有问题，可以用 `kubectl edit onecloudcluster -n onecloud <cluster-name>`, 把 ".spec.version" 改为 “v3.1.5-20200525.0”, 让里面服务组件的使用统一镜像版本再试试。

**Comment by zexi** [comment]
@lshmouse 可以用微信关注下面的二维码，通过小程序加入技术交流群：

image

**Comment by zexi** [comment]
/lgtm
/approve

Thx!

File: content/zh/docs/quickstart/minikube.md (hunk 1 above)
Comment (lshmouse) [reply content/zh/docs/quickstart/minikube.md:88]: @zexi 通过制定onecloud-operator和onecloud-cluster 版本， example-onecloud-cluster-notify/example-onecloud-cluster-monitor Pod启动问题解决了，多谢～ 剩余两个Pod还是失败
- example-onecloud-cluster-host-deployer
 ```
Successfully assigned onecloud/example-onecloud-cluster-host-deployer-rqkbh to onecloud
MountVolume.SetUp failed for volume "cloud" : hostPath type check failed: /opt/cloud is not a directory
Unable to attach or mount volumes: unmounted volumes=[cloud etc-yunion usr], unattached volumes=[sys default-token-z69qg proc var-run-openvswitch config certs dev cloud run tmp var-run-vmware etc-yunion var-log-openvswitch usr]: timed out waiting for the condition
MountVolume.SetUp failed for volume "usr" : hostPath type check failed: /usr/local is not a directory
MountVolume.SetUp failed for volume "etc-yunion" : hostPath type check failed: /etc/yunion is not a directory
 ```

- example-onecloud-cluster-autoupdate
```
[I 200630 22:38:04 policy.(*SPolicyManager).start(policy.go:152)] PolicyManager start to fetch policies ...
[F 200630 22:38:04 versions.InitDriver(versions.go:548)] Init kubernetes driver: failed to get onecloud cluster onecloud/default: onecloudclusters.onecloud.yunion.io "default" not found
```

Comment (lshmouse) [reply content/zh/docs/quickstart/minikube.md:88]: @zexi 内部原理不是很了解，请多指教～

**Pull request merged**